### DISCUSSION TITLE
Plugins catalog for older versions

### DISCUSSION THREAD

**Discussion by diogofd8** [apache/netbeans#8090] [General]
I never stopped using NetBeans 8.2. What works, works, I guess. However, I've noticed that the plugins are not longer available.

https://cwiki.apache.org/confluence/display/NETBEANS/Where+to+download+plugins+for+NetBeans+10.0+and+earlier

Typically one would go here and get the catalogs from librebeans.org, however, those links are down. Is there any new alternative? Are those gone forever?

**Comment by Chris2011** [comment]
Hey @diogofd8, I updated the confluence page, it is now this url: https://plugins.archive.openbeans.org/catalogue/11.0/catalog.xml. Also thx to @emilianbold to restart the machine.
